src/main/java/org/spongepowered/common/config/type/GlobalConfig.java

```diff
@@ -58,46 +58,51 @@
     @Setting(comment = "Configuration options related to the Sql service, including connection aliases etc")
     private SqlCategory sql = new SqlCategory();
 
-    @Setting
+    @Setting(comment = "Configuration options related to commands, including command aliases and hidden commands.")
```

Comment:
We're being very inconsistent here between "Configuration options" and just "Options" - choose one or the other please.

Reply:
Fixed